README.md

```diff
@@ -86,10 +86,10 @@ From PyPI:
     $ kafka-check --cluster-type sample_type min_isr
 ```
 
-### Check number of under replicated partitions
+### Check number of unavailable replicas
 
 ```shell
-    $ kafka-check --cluster-type sample_type under_replicated
+    $ kafka-check --cluster-type sample_type replica_availability
```

Comment:
replica_availability makes me think  how many replicas are available, instead of no. unavailable replicas which this command actually represent. We should probably change to reflect the true intent. What do you think?